src/RA_AchievementOverlay.h

```diff
@@ -157,43 +144,46 @@ class AchievementOverlay
 public:
     struct NewsItem
     {
-        unsigned int m_nID;
+        unsigned int m_nID{};
         std::string m_sTitle;
         std::string m_sPayload;
-        time_t m_nPostedAt;
+        time_t m_nPostedAt{};
         std::string m_sPostedAt;
         std::string m_sAuthor;
         std::string m_sLink;
         std::string m_sImage;
     };
 
 private:
-    int	m_nAchievementsScrollOffset;
-    int	m_nFriendsScrollOffset;
-    int	m_nMessagesScrollOffset;
-    int	m_nNewsScrollOffset;
-    int	m_nLeaderboardScrollOffset;
-
-    int	m_nAchievementsSelectedItem;
-    int	m_nFriendsSelectedItem;
-    int	m_nMessagesSelectedItem;
-    int	m_nNewsSelectedItem;
-    int	m_nLeaderboardSelectedItem;
-
-    mutable int m_nNumAchievementsBeingRendered;
-    mutable int m_nNumFriendsBeingRendered;
-    mutable int m_nNumLeaderboardsBeingRendered;
-
-    BOOL					m_bInputLock;	//	Waiting for pad release
-    std::vector<NewsItem>	m_LatestNews;
-    TransitionState			m_nTransitionState;
-    float					m_fTransitionTimer;
-
-    OverlayPage				m_Pages[5];
-    unsigned int			m_nPageStackPointer;
-
-    ra::services::ImageReference m_hOverlayBackground;
-    ra::services::ImageReference m_hUserImage;
+    static _CONSTANT_VAR PAGE_TRANSITION_IN{ -0.2F };
+    static _CONSTANT_VAR PAGE_TRANSITION_OUT{ 0.2F };
+
+    int	m_nAchievementsScrollOffset{};
+    int	m_nFriendsScrollOffset{};
+    int	m_nMessagesScrollOffset{};
+    int	m_nNewsScrollOffset{};
+    int	m_nLeaderboardScrollOffset{};
+
+    int	m_nAchievementsSelectedItem{};
+    int	m_nFriendsSelectedItem{};
+    int	m_nMessagesSelectedItem{};
+    int	m_nNewsSelectedItem{};
+    int	m_nLeaderboardSelectedItem{};
+
+    mutable int m_nNumAchievementsBeingRendered{};
+    mutable int m_nNumFriendsBeingRendered{};
+    mutable int m_nNumLeaderboardsBeingRendered{};
+
+    BOOL					m_bInputLock{};	//	Waiting for pad release
+    std::vector<NewsItem>	m_LatestNews{};
+    TransitionState			m_nTransitionState{};
+    float					m_fTransitionTimer{ PAGE_TRANSITION_IN };
+
+    std::array<Page, 5>		m_Pages{ Page::Achievements };
+    unsigned int			m_nPageStackPointer{};
+
+    ra::services::ImageReference m_hOverlayBackground{ ra::services::ImageType::Local, "Overlay\\overlayBG.png" };
+    ra::services::ImageReference m_hUserImage{ ra::services::ImageType::UserPic, RAUsers::LocalUser().Username() };
```

Comment:
This way allows us to not have to define any constructors explicitly. Copying will be deleted in another PR but moving will be kept. 

The global is reinitialized using a move assign in `RA_Core.cpp (InitCommon)` after the image repository is initialized. It had to "reinitizlied" because the `AchievementOverlay `global is may or may not be initialized before the `ImageRepository` global.

The "`Initialize`" method is gone because it's initialized during construction.

Reply:
Using `Username` in this manner seems dangerous. I expect the `g_AchievementOverlay` instance to be initialized before we're logged in.

Reply:
Oh, I see what you're doing. "reinitializing" seems like a hack. Can you just add an `UpdateImages()` method that resets these two fields instead of the entire object?